antwar.config.js

```diff
@@ -89,7 +89,8 @@ module.exports = {
           false,
           /^\.\/.*\.md$/
         );
-      }
+      },
+      redirects: {'external-config': 'javascript-alternatives'}
```

Comment:
This is syntactically invalid. Change it so that is is analogue to [here](https://github.com/webpack/webpack.js.org/blob/3b1c80b74e5026f73ecbace8ab0a1a147c8dccc1/antwar.config.js#L68-L70):

```diff
-      },
-      redirects: {'external-config': 'javascript-alternatives'}
+      }, {
+        'external-config': 'javascript-alternatives'
+      }
```